### PR TITLE
Could not find config error fix

### PR DESCRIPTION
**Pull request by alanvardy**

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -58,17 +58,24 @@ impl Config {
         Ok(String::from("✓"))
     }
 
-    pub fn load(path: String) -> Result<Config, String> {
+    pub fn load(path: &str) -> Result<Config, String> {
```
Comment (alanvardy) [inline src/config.rs:61]: I need to have functions borrow string slices more!

File: src/config.rs [hunk 2]
```diff
@@ -168,16 +177,31 @@ pub fn get_or_create(config_path: Option<&str>) -> Result<Config, String> {
...
+
+            // When we move the config file we also need to rename the path in JSON
+            if config.path != path {
+                let new_config = config.set_path(&path);
+                new_config.clone().save()?;
+                Ok(new_config)
+            } else {
+                Ok(config)
+            }
+        }
```
Comment (alanvardy) [inline src/config.rs:193]: I check to see if the path in the JSON differs from the path we looked it up with and edit/resave it if they are not the same.

**Comment by alanvardy** [comment]
Closes #139

**Review by hyblm — APPROVED** [review]

**Review by hyblm — APPROVED** [review]
Looks good

**Pull request merged**